src/ohri-form.scss

```diff
@@ -68,15 +69,3 @@
   padding: 1rem;
 }
 
-aside[class*='-esm-patient-chart__workspace-window__container__'] {
-  border-left: 1px solid #8d8d8d !important;
-}
-
-aside[class*='-esm-patient-chart__workspace-window__container__'] > div {
-  display: flex;
-  bottom: 0;
-}
-
-aside[class*='-esm-patient-chart__workspace-window__container__'] > div > div {
-  flex-grow: 1;
-}
```

Comment:
These global styles were the root cause behind the forms dashboard not rendering correctly in the workspace when a clinical form got launched.